mtp_noms_ops/assets-src/stylesheets/views/_job-information-form.scss

```diff
@@ -0,0 +1,11 @@
+.mtp-job-information-form {
+    .mtp-conditional-wrapper {
+        @extend .panel;
```

Comment:
hopefully we can rip this out once we move noms-ops to the gds design system. this is duplication of `.mtp-security-advanced-search .mtp-conditional-wrapper`

Reply:
Yep, it could have been dropped in there but thought it better to keep this separate (doing the same for the JS looked more problematic though).